addons/mail/static/src/models/dropdown_menu_view.js

```diff
@@ -0,0 +1,181 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, many, one } from '@mail/model/model_field';
+import { clear } from '@mail/model/model_field_command';
+
+registerModel({
+    name: 'DropdownMenuView',
```

Comment:
I don't like the name of this component, and the split in general.

It does 2 things at the same time: popover, and handles selection in a list.

I would expect to see 2 different components: popover (we already have) and "navigable list" (with main and extra).

lifecycleHooks, contains, _onClickCaptureGlobal are duplicate of popover code